.github/workflows/package-publish.yml

```diff
@@ -50,17 +50,6 @@ jobs:
         run: |
           git config --global user.name "sendbird-sdk-deployment"
           git config --global user.email "sha.sdk_deployment@sendbird.com"
-          echo "RELEASE_DATE=$(date +'%b %d %Y')" >> $GITHUB_ENV
-          echo "LATEST_TAG=$(git describe --tags $(git rev-list --tags --max-count=1))" >> $GITHUB_ENV
-      - name: Generate CHANEGLOG
```

Comment:
This part doesn't really generate a better CHANGELOG than I do 😅 
I'm thinking of another tool to do this.